zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -467,10 +447,15 @@ def test_blacklist(self):
         )
 
         # Attempt to add device with existent path should succeed
+        unused_dev = self.get_unused_device(unit_name)
+        if unused_dev is None:
+            raise unittest.SkipTest(
+                "Skipping rest of test because there are no unused devices")
+
         action_obj = zaza_model.run_action(
             unit_name=unit_name,
             action_name='blacklist-add-disk',
-            action_params={'osd-devices': '/dev/vda'}
```

Comment:
I don't believe that this device is expected to be unused - on serverstack, vda will be mounted at /!

Reply:
right. it's "unused" from ceph's perspective. I guess "pristine" would be a better name?

Reply:
generally speaking, we want it to be an in-use device that's not used by Ceph, the version using /dev/vda is using the operating system's root disk